translate/filters/test_checks.py

```diff
@@ -592,6 +592,10 @@ def test_printf():
     assert fails(stdchecker.printf, "I am %#100s", "Ek is %10s")
     assert fails(stdchecker.printf, "... for user %.100s on %.100s:", "... lomuntu osebenzisa i-%. I-100s e-100s:")
     assert passes(stdchecker.printf, "%dMB", "%d MG")
+    assert fails(stdchecker.printf, "Discount 10% something", "Korting tien procent")
+    assert fails(stdchecker.printf, "Discount 10%%", "Koring 10%")
+    assert fails(stdchecker.printf, "%%$", "%$")
```

Comment:
Perhaps I am looking at this the wrong way, but when a `%%` literal is present the `printf` should just skip it and don't consider it a failure. So this two tests above should be passing.